app/services/notifications/moderation/send.rb

```diff
@@ -0,0 +1,39 @@
+# Send notifications from moderation
+module Notifications
+  module Moderation
+    class Send
+      def initialize(available_moderators, notifiable)
+        @available_moderators = available_moderators
+        @notifiable = notifiable
+      end
+
+      def self.call(*args)
+        new(*args).call
+      end
+
+      delegate :user_data, :comment_data, to: Notifications
+
+      def call
+        # notifiable is currently only comment
+        moderator = available_moderators.sample
+        dev_account = User.dev_account
+        json_data = {
+          user: user_data(dev_account)
```

Comment:
can save one line here :D `user_data(User.dev_account)`

Reply:
I have done that.